get-options.sh

```diff
@@ -20,7 +20,7 @@ error() {
 
 show_help() {
 	echo "
-         usage: ./run.sh --platform core.x|xeon|dgpu.x --inputsrc RS_SERIAL_NUMBER|CAMERA_RTSP_URL|file:video.mp4|/dev/video0 [--classification_disabled] [ --ocr_disabled | --ocr [OCR_INTERVAL OCR_DEVICE] ] [ --barcode_disabled | --barcode [BARCODE_INTERVAL] ] [--realsense_enabled] --workload dlstreamer|ovms
+         usage: ./run.sh --platform core.x|xeon|dgpu.x --inputsrc RS_SERIAL_NUMBER|CAMERA_RTSP_URL|file:video.mp4|/dev/video0 [--classification_disabled] [ --ocr_disabled | --ocr [OCR_INTERVAL OCR_DEVICE] ] [ --barcode_disabled | --barcode [BARCODE_INTERVAL] ] [--realsense_enabled]
```

Comment:
If we are removing DLStreamer pipelines classification_disabled, ocr_disabled, barcode_disabled, ocr, and barcode parameters can be removed. Those are specific to the run-dlstreamer script.

Reply:
true, i can remove those dlstreamer-related commandline args

Reply:
removed and updated